apps/src/templates/sectionAssessments/assessmentDataShapes.js

```diff
@@ -0,0 +1,20 @@
+import { PropTypes } from 'react';
+
+export const studentAnswerDataPropType = PropTypes.shape({
+  id:  PropTypes.string,
+  name: PropTypes.string,
+  studentAnswers: PropTypes.array,
+});
```

Comment:
Are any/all of these propTypes required? same for below

Reply:
Updated studentAnswers to PropTypes.array.isRequired.